WORKING_GROUPS.md

```diff
@@ -16,6 +16,16 @@ If the work defined in a Working Group charter is completed the Working
 Group should be dissolved and the responsibility for governance absorbed
 back in to the CTC.
 
+## Joining a WG
+
+To find out how to join a working group, consult the GOVERNANCE.md in
+the working group's repository, or simply open an issue there.
```

Comment:
Nit: remove `simply` and `there` -> `in the working group's respository` (repetitive but clear)